package.json

```diff
@@ -16,7 +16,9 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "test": "istanbul cover _mocha"
   },
-  "dependencies": {},
+  "dependencies": {
+    "color-convert": "^1.9.1"
```

Comment:
The initial idea of the project was not to use external dependencies. I believe you can check if terminal supports 256 color by looking into environment variables, take a look into term sequences and what you have in env.